lib/Twig/Template.php

```diff
@@ -321,33 +321,6 @@ public function getBlocks()
     }
 
     /**
-     * Returns the template source code.
-     *
-     * @return string|null The template source code or null if it is not available
-     */
-    public function getSource()
```

Comment:
> Do we know whether some people are doing crazy things by implementing a child class of Twig_Template by hand ? We probably don't care about supporting it though

To reply to @stof's question: Yes, eZ Platform does it in their code to support integration with legacy version which used custom templating engine.

https://github.com/ezsystems/LegacyBridge/blob/master/mvc/Templating/Twig/Template.php